aiken-contracts/fractional/README.md

```diff
@@ -55,21 +55,43 @@ pub type Datum {
 }
 ```
 
-The bundle token information needs to match what is on the UTxO and the cost token information can be whatever token the seller wishes to be paid in. The maximum bundle size is a hardcoded maximum of 10 bundles. This is an arbitrary pick and can be changed to whatever is needed in production.
+The bundle token information needs to match what is on the UTxO and the cost token information can be whatever token the seller wishes to be paid in for a completed sale.
```

Comment:
Can a sale take two different tokens to complete a sale? Say... X NEWM + Y Ada?

Reply:
Instead of doing a pid + tkn + amt combo for the cost object we can use a dict that holds a value.